### PR TITLE
src/: fixed error message

### PR DESCRIPTION
**Pull request by joscollin**

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins test make check

**Comment by joscollin** [comment]
jenkins test make check

**Comment by badone** [comment]
I'll take a look at this

File: src/common/ceph_releases.cc [hunk 1]
```diff
@@ -38,7 +38,7 @@ bool can_upgrade_from(ceph_release_t from_release,
   if (cutoff < to_release) {
     err << "recorded " << from_release_name << " "
         << to_integer<int>(from_release) << " (" << from_release << ") "
-        << "is >2 releases older than installed "
+        << "is 2 or more releases older than installed "
```
Comment (tchaikov) [inline src/common/ceph_releases.cc:41]: this is wrong.

Comment (badone) [reply src/common/ceph_releases.cc:41]: Sigh... looks like this should read "is more than two releases older than installed"

**Comment by joscollin** [comment]
retest this please

**Comment by joscollin** [comment]
jenkins test dashboard backend

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-08-08_13:50:13-rados-wip-kefu-testing-2020-08-08-1822-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-08-09_03:30:55-rados-wip-kefu-testing-2020-08-08-1822-distro-basic-smithi/

failure in upgrade test also exists in master

- https://pulpito.ceph.com/kchai-2020-08-09_05:31:11-rados-master-distro-basic-smithi/

**Pull request merged**